### PR TITLE
Add build info for Golang on Alpine linux.

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

**Comment by petrosagg** [comment]
Where is this patch from?

**Comment by nghiant2710** [comment]
@petrosagg the patch is from this commit: https://github.com/golang/go/commit/1439158120742e5f41825de90a76b680da64bf76
and this is the original thread: https://github.com/golang/go/commit/1439158120742e5f41825de90a76b680da64bf76

File: build.sh [hunk 1]
```diff
@@ -20,9 +21,9 @@ function version_le() { test "$(echo "$@" | tr " " "\n" | sort -V | tail -n 1)"
 # in order to build Go 1.5, need to download Go 1.4 first
 if version_le $GOLANG_VERSION "1.5"; then
 	mkdir /go-bootstrap
-	wget http://resin-packages.s3.amazonaws.com/golang/v$GOROOT_BOOTSTRAP_VERSION/go-v$GOROOT_BOOTSTRAP_VERSION-linux-$ARCH.tar.gz
-	tar -xzf "go-v$GOROOT_BOOTSTRAP_VERSION-linux-$ARCH.tar.gz" -C /go-bootstrap --strip-components=1
-	rm go-v$GOROOT_BOOTSTRAP_VERSION-linux-$ARCH.tar.gz
+	wget http://resin-packages.s3.amazonaws.com/golang/v$GOROOT_BOOTSTRAP_VERSION/go$GOROOT_BOOTSTRAP_VERSION.linux-$ARCH.tar.gz
```
Comment (petrosagg) [inline build.sh]: verify checksums here

**Comment by nghiant2710** [comment]
@petrosagg, I added checksums verification and moved commit hash and checksums to a new file

**Comment by petrosagg** [comment]
SGTM

**Pull request merged**